.ask/config

```diff
@@ -0,0 +1,21 @@
+{
+  "deploy_settings": {
+    "default": {
+      "skill_id": "",
+     	"resources": {
+        "lambda": [
+          {
+            "functionName": "",
+            "alexaUsage": [
+              "custom/default"
+            ],
+            "runtime": "python3.6",
```

Comment:
AWS Lambda supports Python 3.7 too [now](https://aws.amazon.com/blogs/compute/python-3-7-runtime-now-available-in-aws-lambda/). So we need to look into supporting that in future.